examples/javascript/simple_transfer/transfer.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * This example shows how to use the Aptos client to create accounts, fund them, and transfer between them.
+ */
+
+const { Account, Aptos, AptosConfig, Network } = require("aptos");
+
+const APTOS_COIN = "0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>";
+const ALICE_INITIAL_BALANCE = 100_000_000;
+const BOB_INITIAL_BALANCE = 100;
+const TRANSFER_AMOUNT = 0;
+
+/**
+ * Prints the balance of an account
+ * @param aptos
+ * @param name
+ * @param address
+ * @returns {Promise<*>}
+ *
+ * TODO: Change to AccountAddress for address
+ */
+const balance = async (aptos, name, address) => {
+  let balance = await aptos.getAccountResource({ accountAddress: address, resourceType: APTOS_COIN });
+
+  let amount = Number(balance.data.coin.value);
+
+  console.log(`${name}'s balance is: ${amount}`);
+  return amount;
+};
+
+const example = async () => {
+  console.log("This example will create two accounts (Alice and Bob), fund them, and transfer between them.");
+
+  // Setup the client
+  const config = new AptosConfig({ network: Network.DEVNET });
+  const aptos = new Aptos(config);
+
+  // Create two accounts
+  let alice = Account.generate({ scheme: 0 });
+  let bob = Account.generate({ scheme: 0 });
```

Comment:
nit: Should we use SigningScheme enum here, instead of directly using 0?

Reply:
The enum only exists in Typescript, so unfortunately, it didn't work!  Had to use the number directly rather than the enum in Javascript.

It's been an interesting learning of Javascript vs Typescript when using them together

Reply:
We may want both the constants and the enum then (since I was the one who pushed to replace the constants with an enum)

Reply:
I like the enum as well, too bad for js i guess lol

Reply:
interesting.... we might want to have consts for js and enums for ts, we can probably build those "types" in separate files so if you use js import x.js